models/DataObject/ClassDefinition/Data/EncryptedField.php

```diff
@@ -291,7 +271,7 @@ public function checkValidity(mixed $data, bool $omitMandatoryCheck = false, arr
      */
     public function synchronizeWithMasterDefinition(Model\DataObject\ClassDefinition\Data $masterDefinition)
     {
-        $this->datatype = $masterDefinition->datatype;
+
```

Comment:
Just double checking .. was the method left empty by intent? If yes, can we just remove it?